Python/Product/Debugger.Concord/ModuleManager.cs

```diff
@@ -160,6 +165,27 @@ public static void PyCode_New(DkmThread thread, ulong frameBase, ulong vframe, u
                 }.SendLower(process);
             }
 
+            public static void _PyCode_New(DkmThread thread, ulong frameBase, ulong vframe, ulong returnAddress) {
+                var process = thread.Process;
+                var cppEval = new CppExpressionEvaluator(thread, frameBase, vframe);
+
+                var filenamePtr = cppEval.EvaluateUInt64("con->filename");
```

Comment:
should we check 'filenamePtr " for null?

Reply:
The FromAddress (line after this) would return a null object I believe, so that check should be redundant.

Yeah it has this code here:

```
 public static PyObject FromAddress(DkmProcess process, ulong address) {
     if (address == 0) {
         return null;
     }
```